code/dqc/refdatacheck.q

```diff
@@ -2,10 +2,10 @@
 
 /- Check whether the referenced column of a table is in another column of another table
 refdatacheck:{[tablea;tableb;cola;colb]
-  .lg.o[`refdatacheck;"checking whether refrence data is covered in the other column"];
+  .lg.o[`refdatacheck;"checking whether reference data is covered in the other column"];
   msg:$[c:all r:tablea[cola]in tableb colb;
```

Comment:
If not using `msg` any more can remove the variable.

Reply:
I believe it is still being used to return the msg at the end of the function, after logging